Source/Public/Publish-Bicep.ps1

```diff
@@ -1,14 +1,30 @@
 function Publish-Bicep {
     [CmdletBinding()]
```

Comment:
parametersets are used, so we should have a defaultparameterset in CmdletBinding()